### PR TITLE
Whitelist discord.gg/xbox from invite auto-warn

### PR DESCRIPTION
**Pull request by FloatingMilkshake**
The Xbox server is verified and people often refer others to it for support. Unfortunately, those who are Tier 1 seem to do this more often than you'd expect, so they're often warned just for sending a link to the Xbox server. Whitelisting it would solve this problem, and it's a trusted server so there shouldn't be any concern about whitelisting it.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
   "homeChannel": 793285213823172628,
   "investigationsChannel": 745276933767430255,
   "warningDaysThreshold": 30,
-  "inviteExclusion": "microsoft",
+  "inviteExclusion": {
```
Comment (Erisa) [inline config.json:16]: This should be [] instead

**Review by Erisa — CHANGES_REQUESTED** [review]
As well as changing to [] the code for this value needs to be changed to accept and check multiple of them - I can do this for you if you'd like.

**Comment by FloatingMilkshake** [comment]
> I can do this for you if you'd like.

Sure, thanks. Sorry, not very good at this haha

**Comment by Erisa** [comment]
No worries, its partially my fault for opting for the lazy way of only using one.
Going to try and sort it out now.

File: config.json (hunk 1 above)
Comment (Erisa) [reply config.json]: Fixed in https://github.com/Erisa/Cliptok/pull/35/commits/f416bd481da3254c8590dc3a8588f865c7496d1e

**Comment by Erisa** [comment]
Should be good now with https://github.com/Erisa/Cliptok/pull/35/commits/f416bd481da3254c8590dc3a8588f865c7496d1e

**Review by Erisa — APPROVED** [review]

**Comment by Erisa** [comment]
Thanks @FloatingMilkshake!

**Pull request merged**